src/test/java/com/cloudant/tests/util/Utils.java

```diff
@@ -1,13 +1,24 @@
 package com.cloudant.tests.util;
 
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
 import com.cloudant.client.api.CloudantClient;
+import com.cloudant.client.api.model.ReplicatorDocument;
+import com.cloudant.client.api.model.Response;
 
 import org.apache.commons.logging.Log;
 
 import java.io.InputStream;
 import java.util.Properties;
+import java.util.UUID;
 
 public class Utils {
+
+    //Timeout in milliseconds
+    private static long TIMEOUT_CONSTANT = 3200;
```

Comment:
Just call the variable `TIMEOUT_MILLISECONDS` then you can avoid the need to comment the units above. Also, remove `CONSTANT` given that's implied. Also the variable should be final.